packages/aws-cdk-lib/aws-sns/lib/topic.ts

```diff
@@ -76,6 +76,16 @@ export interface TopicProps {
    * @default false
    */
   readonly enforceSSL?: boolean;
+
+  /**
+   * The signature version corresponds to the hashing algorithm used while creating the signature of the notifications,
+   * subscription confirmations, or unsubscribe confirmation messages sent by Amazon SNS.
+   *
+   * For more information, see https://docs.aws.amazon.com/sns/latest/dg/sns-verify-signature-of-message.html.
```

Comment:
```suggestion
   * @see https://docs.aws.amazon.com/sns/latest/dg/sns-verify-signature-of-message.html.
```
Let's use the docstring